src/scripts/birthdaysToDatabase.ts

```diff
@@ -0,0 +1,54 @@
+import "../db"; // imported for side effect
+import Tools from "../common/tools";
+import { BirthdayRepository, Birthday } from "../entities/Birthday";
+import { getUserBirthdate } from "../programs/BirthdayManager";
+import { setTimeout } from "timers";
+
+interface JSONBirthday {
+  id: string;
+  date: string;
+}
+
+async function importBirthdaysToDatabase() {
+  const birthdayRepository = await BirthdayRepository();
+  const birthdays = (<unknown>(
+    await Tools.resolveFile("birthdayMembers")
+  )) as JSONBirthday[];
+
+  const existingBirthdays = await birthdayRepository.find();
+  const existingBirthdayUsers = existingBirthdays.map((b) => b.userid);
+
+  const toCreate = birthdays
+    .filter(({ id }) => existingBirthdayUsers.indexOf(id) === -1)
```

Comment:
```suggestion
    .filter(({ id }) => !existingBirthdayUsers.includes(id))
```
Same thing as with the other indexOf; feel free to ignore if there are drawbacks.